weto/src/main/java/fi/uta/cs/weto/util/NotificationManager.java

```diff
@@ -117,7 +326,7 @@ public void run() {
                 }
 
                 try (StringWriter htmlStringWriter = new StringWriter();
-                    StringWriter textStringWriter = new StringWriter()) {
+                     StringWriter textStringWriter = new StringWriter()) {

```

Comment:
Undo extra space (my bad while undoing the bad merge)
```suggestion
                    StringWriter textStringWriter = new StringWriter()) {
```